SharedLibs/TileMapHelper/src/render/TileMapPixiHelper.ts

```diff
@@ -50,6 +50,39 @@ export namespace PixiTileMapHelper {
     return null;
   }
 
+  /**
+   * Split an atlas image into Pixi textures.
+   *
+   * @param atlasTexture The texture containing the whole tile set.
+   * @param columnCount The number of columns.
+   * @param rowCount The number of rows.
+   * @param tileSize The squared tile size.
+   * @returns A textures cache.
+   */
+  export function parseSimpleTileMapAtlas(
+    atlasTexture: PIXI.BaseTexture<PIXI.Resource>,
+    columnCount: number,
+    rowCount: number,
+    tileSize: number
+  ): TileTextureCache {
+    const textureCache = new TileTextureCache();
+    for (let x = 0; x < columnCount; x++) {
+      for (let y = 0; y < rowCount; y++) {
+        const rect = new PIXI.Rectangle(
+          x * tileSize,
+          y * tileSize,
+          tileSize,
+          tileSize
+        );
+
+        const texture = new PIXI.Texture(atlasTexture, rect);
+
+        textureCache.setTexture(rowCount * x + y, texture);
```

Comment:
what's the first parameter, the key?

Reply:
The id of the tile yes, maybe I should make it more obvious

Reply:
I added a comment